jangjuseong/app.js

```diff
@@ -0,0 +1,51 @@
+// third-party module
+require('dotenv').config(); // 환경변수 사용
+
+const express = require('express');
+
+const cors = require('cors');
+
+const morgan = require('morgan');
+
+const { DataSource } = require('typeorm');
```

Comment:
해당 부분에서 공백을 통해 코드라인을 구분하는 것은 불필요해보입니다!

```suggestion
const express = require('express');
const cors = require('cors');
const morgan = require('morgan');
const { DataSource } = require('typeorm');
```